<!--
  1 failure:  :disappointed_rel...
  0 warning: 
  1 messages
  
  DangerID: danger-id-default;
-->

<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>

  :disappointed_relieved: `npm test` failed. See below for details.
  </td>
    </tr>
  </tbody>
</table>



<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>

  ```
> titanium-mobile@7.4.0 test /Users/build/jenkins/workspace/dk_titanium_mobile_PR-10254-FB5QRG4WKSBM5JP2JRUCNCNHX4DISBHNBEKRIGZYMZSE5WJHBBLA
> grunt

Running "appcJs:src:lintOnly" (appcJs) task

Running "eslint:src" (eslint) task

/Users/build/jenkins/workspace/dk_titanium_mobile_PR-10254-FB5QRG4WKSBM5JP2JRUCNCNHX4DISBHNBEKRIGZYMZSE5WJHBBLA/tests/Resources/ti.ui.textarea.addontest.js
  37:11  error  Missing space before value for key 'window'  key-spacing

✖ 1 problem (1 error, 0 warnings)
  1 error, 0 warnings potentially fixable with the `--fix` option.

Warning: Task "eslint:src" failed. Use --force to continue.

Aborted due to warnings.
npm ERR! Test failed.  See above for more details.

```
  </td>
    </tr>
  </tbody>
</table>


<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

--- a/tests/Resources/ti.ui.textarea.addontest.js
+++ b/tests/Resources/ti.ui.textarea.addontest.js
@@ -58,7 +58,10 @@ describe('Titanium.UI.TextArea', function () {
 			keyboardMessageView.add(keyboardMessage);
 			typingView.add(keyboardMessageView);
 			subwin.add(typingView);
-			tabA.open(subwin);
+
+			setTimeout(function () {
+				tabA.open(subwin);
+			}, 1000);
 		});
 
 		tabGroup.open();

--- a/tests/Resources/ti.ui.textarea.addontest.js
+++ b/tests/Resources/ti.ui.textarea.addontest.js
@@ -34,7 +34,7 @@ describe('Titanium.UI.TextArea', function () {
 		});
 
 		tabB = Ti.UI.createTab({
-			window:windowB,
+			window: windowB,
 			title: 'Tab B'
 		});
 

